Sc apiserver metrics (#1867)
kyma-project/test-infra: prow/scripts/resources/README.md

## Overview

The folder Contains files which are directly used by prow pipeline scripts.

## Content

```

> ├── kube-dns-stub-domains-patch.yaml                          # Enable stubdomain build.kyma-project.io and provide Google root DNS servers IPs.

Author: It's not a typo in stubdomain word.

> ├── kube-dns-stub-domains-patch.yaml                          # Enable stubdomain build.kyma-project.io and provide Google root DNS servers IPs.

Author: Other suggestions commited.

├── limitrange-patch.yaml                                     # Increase kyma-system namespace max memory request for containers.

> ├── prometheus-operator-stackdriver-patch.yaml                # Inject stackdriver collector sidecar, set metrics filters, enable scraping stackdriver target.

Reviewer: ```suggestion
├── prometheus-operator-stackdriver-patch.yaml                # Injects the Stackdriver collector sidecar, sets metric filters, and enables scraping Stackdriver target.
```

Reviewer: Scraping?

Author: Scraping is a word used by Prometheus documentation to describe process of collecting metric values from monitored components.

> └── prometheus-operator-additional-scrape-config.yaml         # Additionall scrape config for prometheus oeprator.

Reviewer: ```suggestion
└── prometheus-operator-additional-scrape-config.yaml         # Additionall scrape configuration for Prometheus operator.
```

Reviewer: Please explain scrape to me.

```

> ## Prometheus operator additional scrape config

Reviewer: ```suggestion
## Prometheus operator additional scrape configuration
```

Reviewer: Scrape?

> Prometheus operator expect to have additional scrape config provided as one secret. This secret is appended to the Prometheus scrape config file as it is.

Reviewer: ```suggestion
Prometheus operator expects to have additional scrape configuration provided as a Secret. This Secret is appended to the Prometheus scrape config file.
```

> Additional scrape config allow add scrape targets outside automatic scrape targets discovery mechanisms.

Reviewer: ```suggestion
Additional scrape config allows you to add scrape targets outside automatic scrape targets discovery mechanisms.
```

> It's admin responsibility to provide syntactically correct scrape configuration.

Reviewer: ```suggestion
It is administrator's responsibility to provide syntactically correct scrape configuration.
```

## Overview

The folder contains files that are directly used by Prow pipeline scripts.

## Directory structure

```

> ├── kube-dns-stub-domains-patch.yaml                          # Enable stubdomain build.kyma-project.io and provides Google root DNS servers IPs.

Reviewer: ```suggestion
├── kube-dns-stub-domains-patch.yaml                          # Enables the build.kyma-project.io stubdomain and provides Google root DNS servers IPs.
```

├── limitrange-patch.yaml                                     # Increases the kyma-system Namespace maximum memory request for containers.
├── prometheus-operator-stackdriver-patch.yaml                # Injects the Stackdriver collector sidecar, sets metric filters, and enables scraping Stackdriver target.

> └── prometheus-operator-additional-scrape-config.yaml         # Additionall scrape configuration for Prometheus operator.

Reviewer: ```suggestion
└── prometheus-operator-additional-scrape-config.yaml         # Additional scrape configuration for Prometheus operator.
```

> ├── kube-dns-stub-domains-patch.yaml                          # Enables stubdomain build.kyma-project.io and provides Google root DNS servers IPs.

Reviewer: From this point, the structure is repeated. We can delete it.

Author: done
